### DISCUSSION TITLE
The server provided an invalid message! Content-Length: 0

### DISCUSSION THREAD

**Discussion by thitcher** [jomjol/AI-on-the-edge-device#2894] [General]
### Issue

The device worked fine for 2 Weeks, since today it got really slow and freezes.
Did an update to latest version (without problems), the readings in HA are working, but the web interface is really slow and stuck.
Often I get the message:"The server provided an invalid message!
Content-Length: 0"
Livestream picture and statistics not avaliable, webinterface freezes

**Comment by caco3** [comment]
This issue usually is caused by a weak wifi connection.
It might also be related to the sued SD card.

Please provide more information (RSSI, debug log, setup situation, ...)

**Reply by thitcher** [reply]
Please see the values from HA:

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/430574/416c26cd-afbf-4660-88fc-12bde767d5e3)

At the moment, it shows "no ip address", can´t reach it via webinterface, but in HA it works fine (?)

**Reply by caco3** [reply]
do you use DHCP? can you get the IP form your router?

The values should show up when you restart the device.

**Comment by thitcher** [comment]
No, I use fixed IP (IP binding via my Unifi-Gateway). 
I forced the watermeter to reconnect (over Unifi), just at that moment I got access to the webinterface and HA shows the IP.

Only 1 minute later, the webinterface becomes laggy and did not load correctly (Firefox):
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/430574/e0289607-8ed7-44f8-a7de-a1fda00a3ff1)

If I load the GUI at my smartphone, everything is fine.

HA:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/430574/bc2c51aa-6526-473e-a356-3a0d71acd9b0)

Unifi:

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/430574/b381a7ad-1431-4721-937d-9bf124631534)
